Nodejs/Product/Nodejs/Guids.cs

```diff
@@ -42,6 +42,8 @@ internal static class Guids
         public static readonly Guid NodejsNpmCmdSet = new Guid(NodejsNpmCmdSetString);
         public static readonly Guid NodeToolsWorkspaceCmdSet = new Guid(NodeToolsWorkspaceCmdSetString);
 
+        public static readonly Guid MigrateToJspsCmdSet = new Guid("5F602F6D-4592-448A-B321-2ECFA442BB29");
```

Comment:
Should update this to `new Guid(MigrateToJspsCmdSetString)` pattern for consistency with the rest of the file.

Reply:
done